opts/network.go

```diff
@@ -0,0 +1,100 @@
+package opts
```

Comment:
@vdemeester opened a pull request here to move the options; https://github.com/docker/cli/pull/82